spec/core_functions/color/mix/hue_interpolation.hrx

```diff
@@ -0,0 +1,34 @@
+<===> options.yml
+:ignore_for:
+  - libsass
+
+<===>
+================================================================================
+<===> longer/input.scss
+a {b: mix(red, green, hsl longer hue)}
```

Comment:
Since the third argument is `$weight`, these methods should probably be passed by name.

Reply:
fixed